src/service/blockaid/index.ts

```diff
@@ -74,8 +74,16 @@ export class BlockAidService {
         transaction: txXdr,
         account_address: source,
       };
-      const data = await this.blockAidClient.stellar.transaction.scan(body);
-      return { data, error: null };
+      const response = await this.blockAidClient.stellar.transaction
+        .scan(body)
+        .withResponse();
+      const request_id = response.response.headers.get("x-request-id");
```

Comment:
making an update here to grab the request-id from the header. We'll need to pass this on to the reporting endpoint to tell Blockaid what warning we are reporting